react/Badge/Readme.md

```diff
@@ -28,3 +30,26 @@ import ButtonAction from 'cozy-ui/transpiled/react/ButtonAction';
   </p>
 </div>
 ```
+
+
+#### Alignment
+
+Only support `bottom-right` 
+
+
+```
+import Badge from 'cozy-ui/transpiled/react/Badge'
+import Icon from 'cozy-ui/transpiled/react/Icon';
+
+
+
+<div>
+  <p>
+    <Badge 
+      content={ <Icon icon="link" size={10} />} type="normal" alignment="bottom-right" size='medium'>
+      <Icon icon="cloud" size="32" />
```

Comment:
Could we use a `ButtonAction ` here as well? I find it's a bit easier to understand which one the badge is and which one the other component is if we don't use Icon for both :)

Reply:
It was to have a really life example since we currently don't use Badge with a ButtonAction

Reply:
Mmh ok fair enough!